runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowPipelineRegistrar.java

```diff
@@ -34,28 +33,26 @@
   private DataflowPipelineRegistrar() { }
 
   /**
-   * Register the {@link DataflowPipelineOptions} and {@link BlockingDataflowPipelineOptions}.
+   * Register the {@link DataflowPipelineOptions}.
    */
   @AutoService(PipelineOptionsRegistrar.class)
   public static class Options implements PipelineOptionsRegistrar {
     @Override
     public Iterable<Class<? extends PipelineOptions>> getPipelineOptions() {
       return ImmutableList.<Class<? extends PipelineOptions>>of(
-          DataflowPipelineOptions.class,
-          BlockingDataflowPipelineOptions.class);
+          DataflowPipelineOptions.class);
     }
   }
 
   /**
-   * Register the {@link DataflowRunner} and {@link BlockingDataflowRunner}.
+   * Register the {@link DataflowRunner}.
    */
   @AutoService(PipelineRunnerRegistrar.class)
   public static class Runner implements PipelineRunnerRegistrar {
     @Override
     public Iterable<Class<? extends PipelineRunner<?>>> getPipelineRunners() {
       return ImmutableList.<Class<? extends PipelineRunner<?>>>of(
-          DataflowRunner.class,
-          BlockingDataflowRunner.class);
+          DataflowRunner.class);
```

Comment:
May be worth a follow-up to modify PipelineRunnerRegistrar to only return a single PipelineRunner. Definitely not necessary in this PR.